bindings/python/gnucash_core.py

```diff
@@ -46,6 +47,29 @@
     gnc_numeric_create, double_to_gnc_numeric, string_to_gnc_numeric, \
     gnc_numeric_to_string
 
+try:
+    import gettext
+    # install gettext for _-function, needs path to locales
+    # ToDo: this replicates gettext init in gnucash-bin.c
+    # - is it possible to get direct access ? It doesn't seem so.
+    # - if that's not possible mimic the whole init process.
+    #   That should be the case now, as win32 seems irrelevant
+    #   here as the python parts only work on linux.
```

Comment:
It's not possible, Python's gettext separately wraps and links libintl so there's no way to share the initialization.  You can remove the ToDo comment.

Reply:
1c22ca3